lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -94,6 +94,23 @@ cl::opt<bool> SPIRVAllowUnknownIntrinsics(
     cl::desc("Unknown LLVM intrinsics will be translated as external function "
              "calls in SPIR-V"));
 
+enum FPContract {
+  FPContractOn,
+  FPContractOff,
+  FPContractFast,
+};
+
+cl::opt<FPContract> FPContractMode(
+    "ffp-contract", cl::desc("Fused FP operations:"), cl::init(FPContractOn),
```

Comment:
All existing translator options are prefixed with `spirv`, but at the same time, I'm not sure that `spirv-fp-contract` looks good

Reply:
I'm not sure either, so I just copied the naming from Clang.

Reply:
`--spirv-fp-contract` looks good to me.
Also I'd change description to something like "Force FP contraction mode"

Reply:
Done.